.github/workflows/run-host-tests.yml

```diff
@@ -51,14 +57,19 @@ jobs:
             # The sdkconfig.ci.linux specifies Linux as the build target with apropriate settings.
             cp sdkconfig.ci.linux sdkconfig.defaults
             idf.py build
-            ./build/${{inputs.app_name}}.elf -r junit -o junit.xml
```

Comment:
This is a reusable action, correct? Wouldn't we miss the junit outputs for the websocket and mdns host tests?

Reply:
@david-cermak good point. I will modify and fix it in another PR. I'll address and resolve it in a subsequent pull request. As today I noticed a problem with the package upgrade for the mDNS test, but I believe the second commit in this pull request will fix the issue. 
Thank you for your review!